website/pages/docs/k8s/installation/multi-cluster/vms-and-kubernetes.mdx

```diff
@@ -0,0 +1,293 @@
+verify_server_hostname = true
+ports {
+  https = 8501
+  http = -1
+  grpc = 8502
+}
+```
+
+## Kubernetes as the Secondary
+
+If you're running your primary datacenter on VMs then you'll need to manually
+construct the [Federation Secret](#federation-secret) in order to federate
+Kubernetes clusters as secondaries.
+
+-> Your VM cluster must be running mesh gateways, and have mesh gateway WAN
+federation enabled. See [WAN Federation via Mesh Gateways](/docs/connect/wan-federation-via-mesh-gateways).
+
+You'll need:
+
+1. The root certificate authority cert placed in `consul-agent-ca.pem`.
+1. The root certificate authority key placed in `consul-agent-ca-key.pem`.
+1. The IP addresses of the mesh gateways running in your VM datacenter.
+1. If ACLs are enabled you must create an ACL replication token with the following rules:
+
+```hcl
+acl = "write"
+operator = "write"
+agent_prefix "" {
+  policy = "read"
+}
+node_prefix "" {
+  policy = "write"
+}
+service_prefix "" {
+  policy = "read"
+  intentions = "read"
+}
+```
+
+This token is used for ACL replication and for automatic ACL management in Kubernetes.
+
+If you're running Consul Enterprise you'll need the rules:
+
+```hcl
+acl = "write"
+operator = "write"
+agent_prefix "" {
+  policy = "read"
+}
+node_prefix "" {
+  policy = "write"
+}
+namespace_prefix "" {
+  service_prefix "" {
+    policy = "read"
+    intentions = "read"
+  }
+}
+```
+
+1. If gossip encryption is enabled, you'll need the key.
+
+With that data ready, you can create the Kubernetes federation secret:
+
+```sh
+kubectl create secret generic consul-federation \
+    --from-literal=caCert=$(cat consul-agent-ca.pem) \
+    --from-literal=caKey=$(cat consul-agent-ca-key.pem)
+    # If ACLs are enabled uncomment.
+    # --from-literal=replicationToken="<your acl replication token>" \
+    # If using gossip encryption uncomment.
+    # --from-literal=gossipEncryptionKey="<your gossip encryption key>"
+```
+
+Then use the following Helm config file:
+
+```yaml
+global:
+  name: consul
+  image: consul:1.8.0-beta1
+  datacenter: dc2
+  tls:
+    enabled: true
+    caCert:
+      secretName: consul-federation
+      secretKey: caCert
+    caKey:
+      secretName: consul-federation
+      secretKey: caKey
+
+  # Delete this acls section if ACLs are disabled.
+  acls:
+    manageSystemACLs: true
+    replicationToken:
+      secretName: consul-federation
+      secretKey: replicationToken
+
+  federation:
+    enabled: true
+
+  # Delete this gossipEncryption section if gossip encryption is disabled.
+  gossipEncryption:
+    secretName: consul-federation
+    secretKey: gossipEncryptionKey
+
+connectInject:
+  enabled: true
+meshGateway:
+  enabled: true
+server:
+  extraConfig: |
+    {
+      "primary_datacenter": "<your VM datacenter name>",
+      "primary_gateways": ["<ip of your VM mesh gateway>", "<other ip>", ...]
+    }
+```
+
+**NOTE: ** You must fill out the `server.extraConfig` section with the datacenter
```

Comment:
Should we use the -> callout box here as in other places (I'm only looking at raw source so maybe there's a good reason visually not to?).

Reply:
At least in GitHb the `**` after the space confuses the markdown parser and puts this whole section in bold :) probably fine on the actual site though!